transformers4rec/torch/utils/padding.py

```diff
@@ -77,15 +77,15 @@ def pad_batch(batch: Batch, padding_lengths: Dict[str, int]) -> Batch:
     ValueError
         If ragged column found with no padding length provided
     """
-    if batch is None or not isinstance(batch, dict):
-        return batch
-
     batch_padded = {}
-    for col_name, col in TensorTable(batch).items():
```

Comment:
TensorTable is not currently compatible with `torch.jit.script` compilation. 

example of one of the errors that shows up (I don't think it prints out all errors, only the first it encounters -> there may be more unsupported things apart from the below example)

```
E   torch.jit.frontend.UnsupportedNodeError: SetComp aren't supported:
E     File "/workspace/merlin/core/merlin/table/tensor_table.py", line 61
E       def _validate_columns(self, cols_dict):
E           col_types = {type(col_obj) for col_obj in cols_dict.values()}
E                       ~ <--- HERE
E           if len(col_types) >= 2:
E               raise TypeError(
E   '__torch__.merlin.table.tensor_table.TensorTable' is being compiled since it was called from 'pad_batch'
```